src/backend/transcriber/src/main/java/com/amazonaws/transcriber/App.java

```diff
@@ -78,8 +77,8 @@ private static void addShutdownHook(Transcriber transcriber, Encoder encoder, Li
             logger.info("Stopping ffmpeg encoding.");
             encoder.stop();
 
-            logger.info("Persisting lifecycle stage TERMINATING in DynamoDb...");
-            lcp.transcriptionTerminating();
```

Comment:
when will TERMINATING state be written for the orchestrator to handle?

Reply:
answered above! assume we'll have something responding to someone hitting stop on the ui